src/controller/subtitle-stream-controller.ts

```diff
@@ -189,6 +189,13 @@ export class SubtitleStreamController
 
       // Clear internal buffered lists
       this.tracksBuffered.forEach((_, index, tracks) => tracks[index] = []);
+
+      if (track.mode === 'showing') { 
+        track.mode = 'hidden';
+        setTimeout(() => {
+            track.mode = 'showing';
+        }, 10);
+      }
```

Comment:
The playback adapters have a method specifically for this use case `clearText()`, which was in fact originally introduced to fix an issue on Xbox with captions staying on screen during ads. We should try calling this function on seeks.

Reply:
Ok. Let me try `clearText()` in case if that works